chalicelib/lib/slack.py

```diff
@@ -37,7 +37,11 @@ def post_message(
         if as_user is not None:
             data["as_user"] = as_user
 
-        data["blocks"] = self.blocks if self.blocks else None
+        if self.blocks:
+            data["blocks"] = self.blocks[:49]
+            if len(self.blocks) > 49:
+                data["text"] += " *Too much data to display, some data is missing"
+                data["text"] += ", select a shorter range.*"
```

Comment:
Why not make it shorter?
```suggestion
        if data["blocks"] and len(data["blocks"]) > 49:
                data["text"] += " *Too much data to display, some data is missing"
                data["text"] += ", select a shorter range.*"
```

Reply:
This won't really work as `data["blocks"]` will not be populated with data. Pushed another rewrite. Does that one make more sense?